editions/tw5.com/tiddlers/filters/examples/keep.tid

```diff
@@ -0,0 +1,32 @@
+context: home work shopping errands
+created: 20160117112509803
+modified: 20160814073039238
+status: maybe todo active done
+tags: [[Operator Examples]] [[keep Operator]] slide
+title: keep Operator (Examples)
+
+<<.using-days-of-week>>
+
+;Keep only those items from a list that occur in the reference list
+<<.operator-example 1 "[list[Days of the Week]] +[keep[Mon Tuesday Wed Thursday Fri ]]">>
+
```

Comment:
`[keep[Mon Tuesday Wed Thursday Fri ]]` the space at the end. ... has it a meaning? just for readability or a typo?